src/app/tests.py

```diff
@@ -1543,3 +1549,25 @@ def test_no_license_id_on_license_requests_edit_xml(self):
         self.assertEqual(resp.status_code,200)
         self.assertIn("app/license_namespace_requests.html",(i.name for i in resp.templates))
         self.assertEqual(resp.resolver_match.func.__name__,"licenseNamespaceRequests")
+
+class TestCronJob(TestCase):
+    def test_delete_old_files(self):
+        """Check if the files older than 10 days are getting deleted or not"""
+        # create a test directory with some files
+        test_dir = os.path.join(os.getcwd() + '/app/media', 'AnonymousUser')
+        os.makedirs(test_dir, exist_ok=True)
+        for i in range(1, 11):
+            file_path = os.path.join(test_dir, f'test_file_{i}.txt')
```

Comment:
Better to store this in array and directly compare it they exists or not. Creating `f'test_file_{i}.txt'` filename like this should be done at one place so that maintenance is easy and can changed quickly.

Reply:
Like this is for the test files only, we are first creating the 10 test files and then based on whether they were deleted or not, we are asserting, so should I direct assert with an array at the end instead of the for loop?